frontend/micro-ui/web/micro-ui-internals/packages/modules/Estimate/src/pages/employee/ViewEstimate.js

```diff
@@ -0,0 +1,101 @@
+import React,{ Fragment,useState,useEffect } from 'react'
+import { Loader,WorkflowActions,WorkflowTimeline,Header,StatusTable,Card,Row,HorizontalNav} from '@egovernments/digit-ui-react-components';
+import { useTranslation } from "react-i18next";
+import ApplicationDetails from '../../../../templates/ApplicationDetails';
+
+const ViewEstimate = (props) => {
+    
+    const { t } = useTranslation()
+    const { tenantId, estimateNumber } = Digit.Hooks.useQueryParams();
+    
+    const businessService = Digit?.Customizations?.["commonUiConfig"]?.getBusinessService("estimate")
+    const [cardState,setCardState] = useState({})
+    const [activeLink, setActiveLink] = useState("Estimate_Details");
+    const configNavItems = [
+        {
+            "name": "Project_Details",
+            "code": "WORKS_PROJECT_DETAILS",
+            "active": true
+        },
+        {
+            "name": "Estimate_Details",
+            "code": "WORKS_ESTIMATE_DETAILS",
+            "active": true
+        }
+    ]
+
+    const ViewEstimate = Digit.ComponentRegistryService.getComponent("ViewEstimatePage");
+    const ViewProject = Digit.ComponentRegistryService.getComponent("ViewProject");
+
+    //fetching estimate data
+    const { isLoading: isEstimateLoading,data:estimate } = Digit.Hooks.estimates.useEstimateSearch({
+        tenantId,
+        filters: { estimateNumber }
+    })
+    //fetching project data
+    const { isLoading: isProjectLoading, data: project } = Digit.Hooks.project.useProjectSearch({
+        tenantId,
+        searchParams: {
+            Projects: [
+                {
+                    tenantId,
+                    id:estimate?.projectId
+                }
+            ]
+        },
+        config:{
+            enabled: !!(estimate?.projectId) 
+        }
+    })
+
+
+    
+    useEffect(() => {
+      //here set cardstate when estimate and project is available
+      setCardState({
+        estimateNumber:estimate?.estimateNumber,
+        estimateType:"Original Estimate",
+        projectId:project?.projectNumber,
+        dateOfProposal: Digit.DateUtils.ConvertEpochToDate(project?.additionalDetails?.dateOfProposal),
+        projectName:project?.name,
+        projectDesc: project?.description
+      }) 
+    }, [project])
+    
+    
+
+    if(isProjectLoading || isEstimateLoading) return <Loader />
+
+    return (
+        <div className={"employee-main-application-details"}>
+            <div className={"employee-application-details"} style={{ marginBottom: "15px" }}>
+                <Header styles={{ marginLeft: "0px", paddingTop: "10px", fontSize: "32px" }}>{t("ESTIMATE_VIEW_ESTIMATE")}</Header>
+            </div>
+            <Card className={"employeeCard-override"} >
```

Comment:
can you convert into a reusable component 
a small view card , which accepts a t , list of label and text pairs

Reply:
Created this component ViewDetailsCard.js